CONTRIBUTING.md

```diff
@@ -121,6 +127,15 @@ the [Google Python Style guide](http://google.github.io/styleguide/pyguide.html)
 project code is examined using `pylint`, `flake8`, `mypy` and `black`, which will be run
 automatically in CI.
 
+### Docstring
+
+The docstrings should follow the [`Comments and Docstrings` section](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings) of Google Python Style guide. We will include a pylint plugin called [docparams](https://github.com/PyCQA/pylint/blob/main/pylint/extensions/docparams.rst) to validate the parameters of docstrings:
```

Comment:
Mention that all public methods require docstring and type annotation. It is recommended to add docstring for all functions.